pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -158,6 +158,9 @@ type ClusterSpec struct {
 	// ExposeStrategy is the strategy used to expose a cluster control plane.
 	ExposeStrategy ExposeStrategy `json:"exposeStrategy"`
 
+	// Optional: APIServerAllowedIPRanges is a list of whitelisted IPs
```

Comment:
Please provide a better description . Also mention it is applicable for the LoadBalancer expose strategy and ignored for others. I would suggest something like:

```
APIServerAllowedIPRanges is a list of IP ranges allowed to access the API server. If not configured, access to the API server is unrestricted. Applicable only if the expose strategy of the cluster is LoadBalancer.
```